### DISCUSSION TITLE
Bug in contributions on profile?

### DISCUSSION THREAD

**Discussion by vitormmatos** [community/community#49520] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi there,

I'm Vítor, I have a profile and I noticed by end of last month not only several contributions on the contribution board disappeared, but also any commit I'm pushing just does not appear as a contribution as well:
https://github.com/vitormmatos

On my settings, private contributions are enabled.

I also used this repository to get github stats on my readme:
https://github.com/jstrieb/github-stats

there were close to 70k lines changed, now it shows 0

Is this a bug or am I missing something?

**Comment by dustinlessard-wf** [comment]
I'm seeing a similar issue where some pull request reviews are not being counted as a contribution on the activity overview despite showing up for that particular day's feed.
<img width="959" alt="Screen Shot 2023-03-14 at 8 39 48 PM" src="https://user-images.githubusercontent.com/2326518/225174157-fa0dce72-3fd6-4399-9378-b62fe5add1f6.png">

https://github.com/dustinlessard-wf?tab=overview&from=2023-03-01&to=2023-03-14

**Reply by 4Furki4** [reply]
I have the same issue, I don't wanna lose my 190-day streak 😢
Besides, the grey gap was green this morning but somehow changed to grey and my streak got broken.
![image](https://github.com/community/community/assets/84590614/50798c75-a521-4f55-8aa1-9ae7b6586e61)

**Reply by vitormmatos** [reply]
Have you changed your profile's email recently?

**Reply by 4Furki4** [reply]
Nope, I've been using the same email since creating my account.

**Reply by omarfaruk-dev** [reply]
> Have you changed your profile's email recently?

Yes, after changing my email taht activity didn't show properly. In this January 2025 i have hundred of activity, but it shows only 25 contributions in 2025. How funny.
How can i solve it? 
![image](https://github.com/user-attachments/assets/a2aa848e-d914-4383-975f-92160e084776)

**Comment by abdessamadpas** [comment]
i have the same issue , there is any solution?

**Comment by vitormmatos** [comment, marked as answer]
I've changed my e-mail used back then and excluded the previous one.
Every commit I've done using the previous e-mail was removed from this counting.

Adding my previous e-mail once again to my Github account, even without setting it as my primary e-mail, was enough for Github to count again those contributions.

Not sure if this count as a workaround or a solution, so I'll re-open this discussion.

**Reply by abdessamadpas** [reply]
yeah the same thing i do

**Reply by PRANAV-SHILU** [reply]
i did same but still not showing after 3 days

**Comment by CesarObedFL** [comment]
i have the same issue, my streak dissapear and i don´t change my email or username at all, my commits are normally displayed at the time they were pushed into the project, but streak dissapeared and the contribution counter does not show them...
is there any solution?
![streak](https://github.com/user-attachments/assets/0e67d48e-1c30-4c91-91a7-25dfe5a664cb)
![streak_counter](https://github.com/user-attachments/assets/ccc8e982-ffac-4a15-8533-f2c94b24fd1a)

this is my last commit on october 13th 2024 at 10:00 pm Mexico City timezone (UTC -06:00)
![commit](https://github.com/user-attachments/assets/82bef6fe-5361-441d-87fd-d42db1c803a5)
